gui-webview/routes/webhooks.js

```diff
@@ -60,19 +60,21 @@ router.post('/', (req, res) => {
     // There may be multiple if batched
     data.entry.forEach((pageEntry) => {
       // Iterate over each messaging event and handle accordingly
-      pageEntry.messaging.forEach((messagingEvent) => {
-        console.log({messagingEvent});
-        if (messagingEvent.message) {
-          receiveApi.handleReceiveMessage(messagingEvent);
-        } if (messagingEvent.postback) {
-          receiveApi.handleReceivePostback(messagingEvent);
-        } else {
-          console.log(
-            'Webhook received unknown messagingEvent: ',
-            messagingEvent
-          );
-        }
-      });
+      if(pageEntry.messaging){
```

Comment:
Can you do 

`if (!pageEntry.messaging) {
  return;
}`

and leave the original code in place? The benefit is easier reading. Also a <space> in between if and (.

BTW this was done as it is since the bot processes only messaging events. But you are right no harm in having the extra check.